moment.js

```diff
@@ -1948,6 +1949,30 @@
     var extendedIsoRegex = /^\s*((?:[+-]\d{6}|\d{4})-(?:\d\d-\d\d|W\d\d-\d|W\d\d|\d\d\d|\d\d))(?:(T| )(\d\d(?::\d\d(?::\d\d(?:[.,]\d+)?)?)?)([\+\-]\d\d(?::?\d\d)?|\s*Z)?)?/;
     var basicIsoRegex = /^\s*((?:[+-]\d{6}|\d{4})(?:\d\d\d\d|W\d\d\d|W\d\d|\d\d\d|\d\d))(?:(T| )(\d\d(?:\d\d(?:\d\d(?:[.,]\d+)?)?)?)([\+\-]\d\d(?::?\d\d)?|\s*Z)?)?/;
 
+    // ietf RFC 2822 regex
+    //	[Group 1: optional][Group 2][Group 3: optional][Group 4]
+    //----
+    // Group 1: "Day[,] "
+    // 	Day= Day of Week ('Mon','Tue','Wed','Thu','Fri','Sat','Sun')
+	// Group 2: "dD Mon [CC]YY HH:MM"
+	// 	dD= Day of Month (1-2-digits) - Strict: 1 to 31 with optional leading zero
+	// 	Mon= Month of Year ('Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec')
+	// 	CC: Century [optional] (2-digits) - Strict: 19 to 99
+	// 	YY: Year in Century (2-digits)
+	// 	HH: Hour of Day (2-digits) - Strict: 00 to 23
+	// 	MM: Minute in Hour (2-digits) - Strict: 00 to 59
+    // Group 3: ":SS "
+    //  SS: Seconds in Minute [optional] (2-digits) - Strict: 00 to 60
+    // Group 4: " (TZ|MIL|TO)"
+    // 	TZ: Timezone ('UT','GMT','EST','EDT','CST','CDT','MST','MDT','PST','PDT')
+    // 	MIL: Military timezone code (A-Z excluding J)
+    // 	TO: Time Offset (+|- 4-digits) - Strict: 0000 to 9959 (as per spec)
+    //====
+	var ietfRegex = new RegExp(/^((?:Mon|Tue|Wed|Thu|Fri|Sat|Sun),?\s)?((?:0?[1-9]|[1-2]?\d|3[01])\s(?:Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s(?:[2-9]\d|19)?\d\d\s(?:[01]\d|2[0-3])		:[0-5]\d)
```

Comment:
Can you pull the day of week and month abbreviations from where they already exist in the English language defaults instead of copying them here?